docs/Z-Score-normalization-script.md

```diff
@@ -5,31 +5,54 @@ For some data types, when uploading to cBioPortal, it is currently necessary to
 Please keep in mind that the z-scores are calculated using only patient data. Hence, overexpressed in this case implies higher expression than the average patient. Also, the source data on which the z-score data is based does not necessarily follow the normal distribution. If your data does not follow the normal distribution, the z-score threshold is less reliable and will result in more false positives or false negatives. You can consider log transforming your value before calculating z-scores to improve this. However, even the logged values may not follow the normal distribution, especially if the data is bimodal.
 
 Currently, the cBioPortal team is woking on a solution that will allow the user to decide whether they want to use zscores, or log values or zscore of log values in the frontend. Furthermore, the team is also working on functionality to be able to use healthy samples in the calculations. 
- 
-### How to proceed
-cBioPortal expects z-score normalization to take place per gene. You can produce this extra z-score file yourself, or let cBioPortal do it for your input files by using `convertExpressionZscores.pl`. Most information that now follows was taken from the comments in `convertExpressionZscores.pl`. We've also added an example of the calculation and example of running the program below.
+
 
 ## The cBioPortal convertExpressionZscores method
```

Comment:
Given the edits in this PR, this header should probably change to something more generic like "Calculating Z-scores"

Reply:
We now have a common header `The cBioPortal Z-Score calculation method` that introduces the two methods and each method is then explained in detail.